src/main/asciidoc/micrometer/influx/main.adoc

```diff
@@ -0,0 +1,93 @@
+[[spring-cloud-data-flow-samples-micrometer-influx-overview]]
+:sectnums:
+:docs_dir: ../..
+
+=== InfluxDB and Grafana
+
+In this demonstration, you will learn how http://micrometer.io[Micrometer] can help to monitor your  http://cloud.spring.io/spring-cloud-dataflow/[Spring Cloud Data Flow] Streams using  https://docs.influxdata.com/influxdb/v1.5/[InfluxDB] and https://grafana.com/grafana[Grafana].
+
+The InfluxDB supports real-time storage of time-series data. It supports downsampling, automatically expiring and deleting unwanted data, as well as backup and restore. Analysis of data is done via a SQL-like query language.
+
+Grafana is open source metrics Dashboard platform.
+
+We will present the steps to prep, configure and rub Spring Cloud Data Flow's `Local` server.
```

Comment:
Rub?  A small description of the arch should be made.

Reply:
+1